### DISCUSSION TITLE
[Bug Report] The claw collider of the robotic arm is not working

### DISCUSSION THREAD

**Discussion by qiangl9618** [isaac-sim/IsaacLab#1150] [Q&A]
HI ,isaaclab developer，I ran into a problem when using the custom import arm and gripper that the gripper collider did not work。

I used my own mechanical arm urdf and the gripper of franka mechanical arm to generate usd file through urdf2usd, but when I used this mechanical arm to grasp objects, I found that the collider of the gripper didn't seem to work, only when the fingertip of the gripper touched the object, there was a collision response. I don't know where the setting problem is, could you please advise me。

[nvdia夹爪穿模.webm](https://github.com/user-attachments/assets/e09c173e-31e2-4a5f-a735-1957218d6e87)

**Comment by kellyguo11** [comment]
Hello, this could be something that requires more tuning on the simulation parameters. Some of the relevant parameters may be around the collision mode and resolution, or the contact offset of the assets. It may also help to increase the position iterations of the simulator.
